src/Documentation/doc/advanced/corporation/faq.md

```diff
@@ -0,0 +1,213 @@
+# FAQ
+
+> What is a corporation good for?
```

Comment:
I don't think `>` (quote) is a good choice to use for the question part of these. Although technically you are "quoting" people's questions, the formatting looks odd (especially in GitHub) and IMO it has the wrong "feel".

I would suggest bullets, or maybe one of the section header types. (`###` or `####`)